### PR TITLE
Fix edge cases for RawStatusString

### PR DESCRIPTION
**Pull request by blegat**
This would produce errors, e.g., when using `JuMP.solution_summary`.

### CODE REVIEW

File: test/MOI_wrapper.jl [hunk 1]
```diff
@@ -184,10 +184,12 @@ end
 
 function test_empty_optimize()
     model = Ipopt.Optimizer()
+    @test MOI.get(model, MOI.RawStatusString()) == "The model has no variable"
```
Comment (odow) [inline test/MOI_wrapper.jl:187]:
```suggestion
    @test MOI.get(model, MOI.RawStatusString()) == "Optimize not called"
```
